introduction.md

```diff
@@ -1,14 +1,21 @@
 # Introduction
 
-This document is intended as a reference for anyone beginning their journey into scientific computing. It may serve as a reference for navigating the command line and C++ guidance, including:
+This document is intended as a reference for anyone beginning their journey into scientific computing. 
+It may serve as a reference for navigating the command line and C++ guidance, including:
 - C++ basics and intermediates
-- Familiarization with POSIX system command line tools and tricks. particularly in High Performance environments and compute clusters
+- Familiarization with POSIX system command line tools and tricks, particularly for High Performance environments and compute clusters
 - Bash shell scripting
 - Project Configuration
-- Version Control with Git, including server side configuration on GitHub
-
-The standard for high performance and scientific computing is C++, therefore, the primary focus of the following sections will be to provide guidance for the C++ programming language. Examples in this document will serve to as a template for configuring and setting up more complicated projects. 
-Let it be noted that the following contents are subjective and not intended to be comprehensive, reflecting personal preferences and conclusions regarding the C++ language; that C++ is a a very large and general purpose language, allowing for freedom of expression and emphasis. To this extent, little importance is placed on concepts such as inheritance, highlighting its relevance simply for applications such as "Curiously Recurring Template Parameters". Furthermore, this document will address modern C++, encompassing language constructs from 2011 and onward, and a review of the standard library..
+- Version Control with Git, including remote server visualization with GitHub
```

Comment:
We still need one more comment discussing remote computing. Something like: Operating complex code bases on computation clusters

Reply:
How is this not addressed by line 6?

Reply:
@ominusliticus  please advise

Reply:
It's not just POSIX, but SLURM, `modules` and MPI ideas